wormhole-connect/src/config/converter.ts

```diff
@@ -106,4 +111,28 @@ export class SDKConverter {
       }
     }
   }
+
+  getTokenIdV2ForSymbol<C extends v2.Chain>(
```

Comment:
Minor nit: Should we rename to `getTokenIdV2ForKey` or something since the `key` is what uniquely identifies a token (`symbol` is a different property)

Reply:
done